AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -119,6 +119,8 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 !
+aeromexico.com#$#html { overflow: auto !important; }
+aeromexico.com#$#.Modal { display: none !important; }
```

Comment:
@gork7777 try to login with this rules

Reply:
I got the problem but how do we separate them from each other?

Reply:
There you should use a scriptlet - fixed it directly https://github.com/AdguardTeam/AdguardFilters/commit/ab535fd1b2cc223f99fe942c15614a147d97e155.